developer_setup.md

```diff
@@ -76,8 +76,8 @@ ManageIQ requires a memcached instance for session caching and a PostgreSQL data
    | ---------- | --- |
    | dnf        | `sudo dnf -y install postgresql-server` |
    | yum        | `sudo yum -y install postgresql-server` |
-   | apt        | `sudo apt -y install postgresql-11` |
-   | brew       | `brew install postgresql@11` |
+   | apt        | `sudo apt -y install postgresql-13` |
```

Comment:
@agrare can you eyeball the debian instructions.  I just replaced numbers.  I hope they follow conventions.

Reply:
I think there is an odd vs even thing here. Don't we want evens?

Reply:
@jrafanie yes that is the convention, but there's no guarantee that v13 will be available on the distribution though.  For example no ubuntu LTS version has the postgresql-13 package, 20.04 has postgresql-12 and 22.04 has postgresql-14.  21.10 does have it but that is out of support already.  Most of the time I think developers just figure out what version they have and use that.  Using 13 here might be fine for an example of what to do, but for a truly copy&paste we'd need to do more

Reply:
never mind ==> https://www.postgresql.org/message-id/200006061607.MAA27510%40candle.pha.pa.us

Reply:
There is a "virtual package" called `postgresql` that points to the specific version which is what would typically be installed by devs.
If we want 13 specifically we'll need to add docs on how to add e.g. the pgdg repository, if we want to work with the version of postgres that the distro has we'll need to modify e.g. the `sudo pg_dropcluster --stop 13 main` below to determine the installed version

Reply:
what do you think makes sense for the most obvious future-proof solution?  It seems weird to continually have to revise these instructions

Reply:
As long as we're okay with the version not being the same that a MIQ appliance might use (these are developer instructions so seems not as important) then the way to go is 100% to use the virtual package.

Something that uses `pg_lsclusters --json | jq -r '.[].version'` to get the version should work.  If you want to change this to just be
```suggestion
   | apt        | `sudo apt -y install postgresql` |
```

I'll comment on the other lines with commands that work dynamically with the latest installed version

Reply:
Homebrew on the mac has the same thing. We used to point to `postgres`

But since on Homebrew, upgrading the version of ssl can trigger an upgrade of postgres to a different major version, it got too fragile. So we nailed down the version.

I would imagine no other packaging system would do this, so just pointing to `postgres` on debian makes sense to me.